src/inject/inject.js

```diff
@@ -14,16 +14,33 @@ chrome.extension.sendMessage({}, function(response) {
 			bar.style.backgroundColor = "dodgerblue";
 			bar.style.textAlign = "center";
 			bar.style.position="fixed"; 
-
 			bar.style.padding = "2px";
 			bar.style.zIndex = "10000";
 			bar.style.borderRadius = "5px";
 
+		var overlay = document.createElement("div");
```

Comment:
Although it looks good, making an overlay over the website makes it unusable. It's too obtrusive in my opinion. I was planning on introducing this enhancement in such a way that all or some divs are blurred using css `filter: blur(1px) grayscale(20%)`

Otherwise if this could be made less obtrusive, i.e. people can scroll and still use the website we could continue using an overlay.